### PR TITLE
Fix isOdd method

### LINKED ISSUE #129: Fix isOdd method
There is currently a method called `isOdd` that lives in `/lib/isOdd.js`.
It is incomplete and needs to be fixed!

### Acceptance Criteria:
1. Running `yarn test isOdd` results in tests passing.
2. You have written a skeleton method for someone else to work on.
3. You have written tests surrounding your skeleton method.
4. Running `yarn lint` does not print any errors to the console!

### Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request! A pull request will most likely be denied if it does not contain a skeleton method for someone else to work on! For more information, please read the [Contributing Guide](https://github.com/BlakeGuilloud/ganon/blob/master/CONTRIBUTING.md).

Thank you so much for your contribution!

### PR DESCRIPTION
**Pull request by abhigyank**
- [x] Running `yarn lint` does not trigger any linter warnings
- [x] The test of the method you have fixed is passing
- [x] You have written a new skeleton method for someone else to work on!

Fix #129

### CODE REVIEW

File: lib/round.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// returns the value of first parameter rounded to its nearest integer
+
+function startsWith(n) {
+  //your code
+}
+
+module.exports = endsWith;
```
Comment (andreasgruenh) [inline lib/round.js]: This should probably be `startsWith`?

Comment (andreasgruenh) [reply lib/round.js]: Ah no I just saw that this should be round and the function name should be adapted as well.

**Comment by abhigyank** [comment]
@andreasgruenh  fixed the issue.

**Comment by andreasgruenh** [comment]
Thanks for contributing!

**Pull request merged**